_posts/2024-03-19-infinispan-operator-2-4-0-release.adoc

```diff
@@ -0,0 +1,45 @@
+---
+layout: blog
+title: "Infinispan Operator 2.4.0"
+permalink: /blog/:year/:month/:day/infinispan-resp-resque-use-case
+date: '2024-03-20T00:00:00.000-00:00'
+author: remerson
+tags: [ "release", "operator", "kubernetes"]
+---
+
+= Infinispan Operator 2.4.0
+
+We're pleased to announce
+https://operatorhub.io/operator/infinispan/stable/infinispan-operator.v2.4.0[version 2.4.0 of the Infinispan Operator]
+for Kubernetes and Red Hat OpenShift.
+
+This is the first Operator release based on Infinispan 15.
+
+== Release highlights:
+
+* Infinispan 14.0.27.Final support
+* Infinispan 15.0.0.Final support
+* Allow Readiness, Liveness and Startup probe values to be configured
+* Simplify JGroups stacks
```

Comment:
I'll add this one on the end

Could we add some more explanations to each part ?